test-apps/display-test-app/package.json

```diff
@@ -67,26 +67,32 @@
     "@itwin/imodels-client-authoring": "^3.0.0",
     "@itwin/imodels-client-management": "^3.0.0",
     "@itwin/map-layers-formats": "workspace:*",
+    "@itwin/object-storage-core": "^2.0.0",
     "@itwin/reality-data-client": "0.9.0",
     "@itwin/webgl-compatibility": "workspace:*",
     "body-parser": "^1.18.2",
     "vhacd-js": "^0.0.1"
   },
   "devDependencies": {
-    "@bentley/react-scripts": "^5.0.0",
+    "@esbuild-plugins/node-globals-polyfill": "^0.2.3",
+    "@esbuild-plugins/node-modules-polyfill": "^0.2.2",
     "@itwin/backend-webpack-tools": "workspace:*",
     "@itwin/build-tools": "workspace:*",
     "@itwin/core-webpack-tools": "workspace:*",
     "@itwin/eslint-plugin": "4.0.0-dev.44",
+    "@rollup/plugin-commonjs": "^25.0.0",
```

Comment:
doesnt seem to be used

Reply:
removed, confirmed still builds and runs normally